### PR TITLE
Refactor to reasonableness

### PR DESCRIPTION
**Pull request by TehShrike**

### CODE REVIEW

**Comment by TehShrike** [comment]
Oh hey, you should flip the Travis switch for this repo

**Comment by TehShrike** [comment]
mm the readme still needs to be updated

File: test/index.js [hunk 1]
```diff
@@ -35,35 +40,33 @@ test('podcast', function(t) {
...
 			subcats: [{
 				text: 'Software',
 				subcats: [{
-					text: 'node.js'
-				}]
-			}]
+					text: 'node.js',
+				}],
+			}],
 		}],
```
Comment (ArtskydJ) [inline test/index.js:54]: Why the trailing commas?

**Comment by ArtskydJ** [comment]
lgtm

If you want to update the readme you can, otherwise I'll merge it and update the readme around the same time.

File: test/index.js (hunk 1 above)
Comment (TehShrike) [reply test/index.js:54]: Mostly all [this stuff](https://dnlserrano.github.io/2015/07/29/a-case-for-trailing-commas/) - personally, I'm to the point where I think I shouldn't have to type the commas at all, a newline should be enough inside of object literals, just like semiclons!

I added a rule to my eslint autoformatter for [dangling commas](https://eslint.org/docs/rules/comma-dangle).

Comment (ArtskydJ) [reply test/index.js:54]: I don't mind them, I was just making sure it was intentional.

The argument that they make git diffs cleaner doesn't really apply in this case :stuck_out_tongue:

Comment (TehShrike) [reply test/index.js:54]: lol, true

**Pull request merged**